www-client/qutebrowser/qutebrowser-1.0.2.recipe

```diff
@@ -65,5 +66,11 @@ INSTALL()
 	export PYTHONPATH=$installLocation:$PYTHONPATH
 	mkdir -p $installLocation
 
+	mkdir -p $appsDir/QuteBrowser
+
 	$python setup.py build $jobArgs install --prefix $prefix
+
+	mv $binDir/qutebrowser $appsDir/QuteBrowser/QuteBrowser
+
+	addAppDeskbarSymlink $appsDir/QuteBrowser/QuteBrowser "QuteBrowser"
```

Comment:
Second argument is only needed in case you want to name a link differently, so
"addAppDeskbarSymlink $appsDir/QuteBrowser/QuteBrowser" is enough